### PR TITLE
Project details (CA-217) + Management page fixes (CA-224) + correct e-mail and thank you messages

### PR DESCRIPTION
**Pull request by logophobia**
This fixes a lot for the project flow. CA-217, CA-224 and the message ron entered in his document. Also fixes project tagging issues during creation (missed that issue, small).

### CODE REVIEW

File: CollAction/Controllers/AdminController.cs [hunk 1]
```diff
@@ -80,7 +85,67 @@ public IActionResult Index()
...
+                bool failed = model.Status == ProjectStatus.Failed && project.Status == ProjectStatus.Running;
+
+                if (approved)
+                {
+                    string approvalEmail = 
+                        $"Hi!<br>" +
+                        $"<br>" +
+                        $"The CollAction Team has reviewed your project proposal and is very happy to share that your project has been approved and now live on www.collaction.org!<br>" +
+                        $"<br>" +
+                        $"So feel very welcome to start promoting it!If you have any further questions, feel free to contact the CollAction Team at collactionteam@gmail.com. And don’t forget to tag CollAction in your messages on social media so we can help you spread the word(FB: @collaction.org, Twitter: @collaction_org)!<br>" +
```
Comment (marcbeermann) [inline CollAction/Controllers/AdminController.cs]: Need to add a whitespace after 'promoting it! If you have'.

File: CollAction/Models/AdminViewModels/ManageProjectViewModel.cs [hunk 2]
```diff
@@ -69,7 +69,7 @@ public class ManageProjectViewModel
 
         [Display(Name = "Hashtag", Prompt = "Max 30 characters. e.g. 'tag1;tag2'")]
         [MaxLength(30)]
-        [RegularExpression(@"^#[a-zA-Z_]\w*(;#[a-zA-Z_]+\w*)*$", ErrorMessage = "No spaces, must contain a letter, can contain digits and underscores. Seperate multiple tags with a colon ';'.")]
+        [RegularExpression(@"^[a-zA-Z_]+(;[a-zA-Z_]+)*$", ErrorMessage = "No spaces, must contain a letter, can contain digits and underscores. Seperate multiple tags with a colon ';'.")]
```
Comment (marcbeermann) [inline CollAction/Models/AdminViewModels/ManageProjectViewModel.cs]: If the intention is to disallow digits in the hashtags then the 'ErrorMessage' should be changed to:
"No spaces or digits are accepted, underscores are allowed. Separate multiple tags with a colon ';'."
N.B. I misspelled 'Separate' in the initial message, that should be corrected also.

File: CollAction/Models/ProjectViewModels/CreateProjectViewModel.cs [hunk 3]
```diff
@@ -81,7 +81,7 @@ public CreateProjectViewModel(SelectList categories, SelectList locations)
 
         [Display(Name = "Hashtag", Prompt = "Max 30 characters. e.g. 'tag1;tag2'")]
         [MaxLength(30)]
-        [RegularExpression(@"^#[a-zA-Z_]\w*(;#[a-zA-Z_]+\w*)*$", ErrorMessage = "No spaces, must contain a letter, can contain digits and underscores. Seperate multiple tags with a colon ';'.")]
+        [RegularExpression(@"^[a-zA-Z_]+(;[a-zA-Z_]+)*$", ErrorMessage = "No spaces, must contain a letter, can contain digits and underscores. Seperate multiple tags with a colon ';'.")]
```
Comment (marcbeermann) [inline CollAction/Models/ProjectViewModels/CreateProjectViewModel.cs]: As above

File: CollAction/Models/ProjectViewModels/EditProjectViewModel.cs [hunk 4]
```diff
@@ -83,9 +83,9 @@ public EditProjectViewModel(SelectList categories, SelectList locations)
         [YouTubeLink]
         public string DescriptionVideoLink { get; set; }
 
-        [Display(Name = "Hashtag", Prompt = "Max 30 characters. e.g. '#tag1;#tag2'")]
+        [Display(Name = "Hashtag", Prompt = "Max 30 characters. e.g. 'tag1;tag2'")]
         [MaxLength(30)]
-        [RegularExpression(@"^#[a-zA-Z_]\w*(;#[a-zA-Z_]+\w*)*$", ErrorMessage = "No spaces, must contain a letter, can contain digits and underscores. Seperate multiple tags with a colon ';'.")]
+        [RegularExpression(@"^[a-zA-Z_]+(;[a-zA-Z_]+)*$", ErrorMessage = "No spaces, must contain a letter, can contain digits and underscores. Seperate multiple tags with a colon ';'.")]
```
Comment (marcbeermann) [inline CollAction/Models/ProjectViewModels/EditProjectViewModel.cs]: As above

File: CollAction/Controllers/AdminController.cs [hunk 5]
```diff
@@ -94,16 +94,16 @@ public IActionResult Index()
...
-                        $"<br>" +
-                        $"Thanks again for driving the CollAction / crowdacting movement!<br>" +
-                        $"<br>" +
-                        $"Warm regards,<br>" +
-                        $"The CollAction team<br>";
+                        "Hi!<br>" +
+                        "<br>" +
+                        "The CollAction Team has reviewed your project proposal and is very happy to share that your project has been approved and now live on www.collaction.org!<br>" +
+                        "<br>" +
+                        "So feel very welcome to start promoting it!If you have any further questions, feel free to contact the CollAction Team at collactionteam@gmail.com. And don’t forget to tag CollAction in your messages on social media so we can help you spread the word(FB: @collaction.org, Twitter: @collaction_org)!<br>" +
```
Comment (marcbeermann) [inline CollAction/Controllers/AdminController.cs]: Need to add a white-space after 'promoting it! If you have'.

File: CollAction/Models/AdminViewModels/ManageProjectViewModel.cs (hunk 2 above)
Comment (logophobia) [inline CollAction/Models/AdminViewModels/ManageProjectViewModel.cs]: Ah, the intention was no longer need to enter "#" when entering tags.

Got very frustrated entering tags, suddenly noticed the need to enter #-es. Bit counter-intuitive. Gonna correct this by allow digits. Would want to allow things allow #savetheforest2015.

We'll need a tag entering UI anyway, this is just an in-between solution.

File: CollAction/Models/ProjectViewModels/DisplayProjectViewModel.cs [hunk 6]
```diff
@@ -35,22 +35,25 @@ public class DisplayProjectViewModel
             }
         }
 
-        public int RemainingDays
-            => Convert.ToInt32(Math.Round((Project.End - Project.Start).TotalDays));
+        public TimeSpan RemainingTime
+            => Project.End - DateTime.UtcNow;
```
Comment (logophobia) [inline CollAction/Models/ProjectViewModels/DisplayProjectViewModel.cs]: Corrected

File: CollAction/Views/Projects/Details.cshtml [hunk 7]
```diff
@@ -136,7 +132,13 @@
             </dd>
```
Comment (logophobia) [inline CollAction/Views/Projects/Details.cshtml]: Done.

**Pull request merged**